tests/test_cliCmdQueryDataIds.py

```diff
@@ -71,12 +72,13 @@ def loadData(self, *filenames: str) -> Butler:
         which should be a YAML import/export file.
         """
         butler = Butler.from_config(self.repo, writeable=True)
+        assert isinstance(butler, DirectButler), "Test expects DirectButler"
         for filename in filenames:
             with open(os.path.join(TESTDIR, "data", "registry", filename)) as stream:
                 # Go behind the back of the import code a bit to deal with
                 # the fact that this is just registry content with no actual
                 # files for the datastore.
-                backend = YamlRepoImportBackend(stream, butler.registry)
+                backend = YamlRepoImportBackend(stream, butler._registry)
```

Comment:
This is because `butler.registry` is no longer a registry but `butler._registry` is?

Reply:
Mostly because I changed `YamlRepoImportBackend` to accept `SqlRegistry` instead of `Registry`. I think that `Registry` at this point exists only because we need backward-compatible interface for `Butler.registry`. When we move all functionality to `Butler` we will eventually remove `Registry` completely.